server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -247,6 +247,70 @@
         "from": [],
         "to": []
       }
+    },
+    {
+      "allow": {
+        "from": [
+          {
+            "kind": "Deployment",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["apiVersion"], 
```

Comment:
The `mutatorRef` path is invalid, willl you ensure the correct ref is present?

Reply:
After discussion, it's confirmed that the mutatorRef path is correct for current implementation.

Reply:
A screenshot, would be great.